packages/mkocaml/mkocaml.1.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+synopsis: "Tool to generate OCaml projects"
+maintainer: "Chris Nevers <christophernevers96@gmail.com>"
+authors: "Chris Nevers <christophernevers96@gmail.com>"
+homepage: "https://github.com/chrisnevers/mkocaml"
+bug-reports: "https://github.com/chrisnevers/mkocaml/issues"
+depends: [
+  "dune" {build}
+  "unix"
```

Comment:
```suggestion
  "base-unix"
```
This is part of the compiler itself. You can notify its use by depending on `base-unix`